client/gui/CGuiHandler.h

```diff
@@ -62,6 +64,7 @@ class CGuiHandler
 	               textInterested;
 
 
+	void handleMouseButtonClick(CIntObjectList &interestedObjs, EIntObjMouseBtnType btn, bool isPressed);

```

Comment:
To be fair it's something I like to know too since it's would be much better if we can add style checking into Travis once and for all. Qt Creator have Beautifier plugin that support 3 different formatters include clang-format, but they all of course require custom configurations since VCMI on VCMI we have somewhat mixed style.

And I never had time to configure formatter.